src/containerbuddy/signals.go

```diff
@@ -68,26 +68,25 @@ func terminate(config *Config) {
 func reloadConfig(config *Config) *Config {
 	signalLock.Lock()
 	defer signalLock.Unlock()
-
+	newConfig, err := loadConfig()
```

Comment:
This moves the log statement to after the attempt to reload the config, whereas I wanted it beforehand so as to signal that's what was happening should there be a problem.